### DISCUSSION TITLE
Mania conversion key count is confusing

### DISCUSSION THREAD

**Discussion by gBasil** [ppy/osu#21941] [Ideas] [answered]
For months, I've had no idea what the osu!mania conversion options were.
<img width="519" alt="image" src="https://user-images.githubusercontent.com/57513430/210040167-dfb4cfff-cf7f-41b3-9bd2-5230f50cf07f.png">
  I realized today that they only apply (or at least to my knowledge seem to) to the mania maps that were converted from other gamemodes, not for maps made specifically for mania. Discussion [#17118](https://github.com/ppy/osu/discussions/17118) suggests the expected behavior, which aligns with what I was expecting. Unless that's on the roadmap, a simple fix would probably be to add some text to the hover that clarifies that it only applies to conversion maps.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/18562
